nano/core_test/node.cpp

```diff
@@ -2562,6 +2564,75 @@ TEST (node, dont_write_lock_node)
 	finished_promise.set_value ();
 }
 
+namespace nano
```

Comment:
Do we need namespace here?

Reply:
Unfortunately yes otherwise it doesn't compile:
https://github.com/google/googletest/blob/3306848f697568aacf4bcca330f6bdd5ce671899/googletest/include/gtest/gtest_prod.h#L55

This class is made a friend inside `nano::active_transactions`